apps/demo/src/app/cart/components/add-to-cart-notification/components/add-to-cart-notification/add-to-cart-notification.component.html

```diff
@@ -1,14 +1,20 @@
 <demo-modal-portal>
-  <daff-modal [show]="open$ | async" (hide)="onHide()" [verticalPosition]="_verticalPosition" [horizontalPosition]="_horizontalPosition">
+  <daff-modal [show]="open$ | async" (hide)="onHide()" [verticalPosition]="_verticalPosition"
+    [horizontalPosition]="_horizontalPosition">
     <div class="add-to-cart-notification">
-      <div *ngIf="!(loading$ | async)" class="add-to-cart-notification__header">
+      <div class="add-to-cart-notification__header"  *ngIf="!(loading$ | async)">
         <span class="add-to-cart-notification__check-icon"><i class="fa fa-check" aria-hidden="true"></i></span>
         <span>Added to Cart</span>
-        <span class="add-to-cart-notification__close"><i (click)="onHide()" class="fa fa-times" aria-hidden="true"></i></span>
+        <span class="add-to-cart-notification__close">
+          <i (click)="onHide()" class="fa fa-times" aria-hidden="true"></i>
+        </span>
       </div>
-      <demo-product-added *ngIf="!(loading$ | async)" class="add-to-cart-notification__product-added" [product]="product$ | async" [qty]="productQty$ | async"></demo-product-added>
+      <demo-product-added *ngIf="!(loading$ | async)" class="add-to-cart-notification__product-added"
+        [product]="product$ | async" [qty]="productQty$ | async"></demo-product-added>
       <div *ngIf="!(loading$ | async)" daff-button-set class="add-to-cart-notification__button-set">
-        <button type="button" daff-stroked-button (click)="onHide()" demoViewCart>View Cart ({{(cartItemCount$ | async)}})</button>
+        <button type="button" daff-stroked-button (click)="onHide()" demoViewCart>View Cart
```

Comment:
View Cart should be on same line as `({{(cartItemCount$ | async)}})`?

Reply:
resolved in b220105547c2ba42232337e89fb5f268dfa12d28